### PR TITLE
The results of ps command should be sorted by create date

### LINKED ISSUE #1334: ps command should sort by create date
The ps command should sort by create date in reverse order.

### PR DESCRIPTION
**Pull request by cgtexmex**
Change will reverse sort results of ps -- newest will appear at the
top of the list.

Fixes #1334

### CODE REVIEW

File: lib/apiservers/engine/backends/container.go [hunk 1]
```diff
@@ -77,6 +78,14 @@ func init() {
 	portMapper = portmap.NewPortMapper()
 }
 
+//TODO: gotta be a better way...
```
Comment (caglar10ur) [inline lib/apiservers/engine/backends/container.go]: nope, that's the best way :)

**Comment by caglar10ur** [comment]
LGTM :sheep: :it:

File: lib/apiservers/engine/backends/container.go (hunk 1 above)
Comment (jzt) [inline lib/apiservers/engine/backends/container.go]: That's The Docker Way™!

**Comment by jzt** [comment]
LGTM

**Comment by cgtexmex** [comment]
REBASE

**Pull request merged**